ui-modules/app-inspector/app/views/main/main.controller.js

```diff
@@ -23,17 +23,19 @@ export const mainState = {
     name: 'main',
     url: '/',
     template: template,
-    controller: ['$scope', '$q', 'brWebNotifications', mainController],
+    controller: ['$scope', '$q', 'brWebNotifications', 'brBrandInfo', mainController],
     controllerAs: 'ctrl'
 };
 
 const savedSortReverse = 'app-inspector-sort-reverse';
 
-export function mainController($scope, $q, brWebNotifications) {
+export function mainController($scope, $q, brWebNotifications, brBrandInfo) {
     $scope.$emit(HIDE_INTERSTITIAL_SPINNER_EVENT);
 
     let ctrl = this;
 
+    ctrl.composerUrl = brBrandInfo.blueprintComposerBaseUrl;
```

Comment:
this felt weird as opposed to `$scope` but was how everything else did it in this file so i continued the pattern